pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1038,6 +1038,16 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListenerForPortOrUDS(l
 		return
 	}
 
+	if listenerOpts.bind == actualWildcard {
+		lops := []*filterChainOpts{}
+		for _, ip := range pluginParams.Node.IPAddresses {
+			lops = append(lops, &filterChainOpts{
```

Comment:
Need comments on this block so that someone else doesn’t nuke it

Reply:
+1 - with bold, blinking text...